scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,151 @@
+# IPC gullwing generator
+
+This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages. (fillet definitions can be changed by pointing to a different definition file.) The script generates footprints in zero orientation version A (pin 1 at top left corner).
+Pads are generated using rounded rectangle pads as suggested in preliminary releases of IPC-7351C.
+
+Examples of supported packages include: QFN, SOIC, SO.
+
+## Running the script
+
+The script requires python 3.4 or newer. Run it with:
+`python3 ipc_gullwing_generator.py size_definitions/eqfp.yaml` (replace `eqfp.yaml` with the size definition file that contains your part.)
+
+### Optional script parameters
+
+* --global_config: the config file defining how the footprint will look like. (KLC)' (default='../../tools/global_config_files/config_KLCv3.0.yaml')
+* --series_config: the config file defining series parameters (footprint naming). (default='../package_config_KLCv3.yaml')
+
+* --density: Density level (L,N,M) (default='N')
```

Comment:
Should this mention it's the IPC density level and not some other generic density parameter? The parameter below, for example, explicitly mentions the relevance to IPC even though the parameter name also includes this word.

Reply:
good point (both here and in python)

Reply:
Were you going to make any change? Such as changing this parameter's description to `IPC 7351 density level`?